### DISCUSSION TITLE
any way we can generate everything required for rust lambdas?

### DISCUSSION THREAD

**Discussion by john-yzaguirre** [projen/projen#2350] [Q&A]
Is this just a generic project?

**Comment by mrgrain** [comment]
Hi @john-yzaguirre There's nothing specific in projen for Rust yet - whether it be a Rust project or Rust lambdas.

If you want to use projen, I reckon your best bet right now is to use a [awscdk-app-ts](https://projen.io/api/API.html#projen-awscdk-awscdktypescriptapp) project and figure it out from there.
